src/components/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import "./Pagination.styles.css";
+
+function Pagination({ page, totalPages }) {
+  const isActive = (index) => {
+    if (index + 1 === page) {
+      return "active";
+    } else {
+      return "";
+    }
+  };
```

Comment:
if the condition is true something is returned, if the condition is false something is returned too. The `if\else` statement could be changed by the ternary op:

```suggestion
  const isActive = index => (index + 1) === page ? "active" : "";
```